agile-development/collaboration/Virtual_Collaboration.md

```diff
@@ -0,0 +1,66 @@
+# Virtual Collaboration and Pair Programming
+
+Pair programming is the de facto work method that most large engineering organizations use for “hands on keyboard” coding. Two programmers, working synchronously, looking at the same screen and attempting to code and design together, which often results in better and clearer code than either could produce individually.
+
+Pair programming works well under the correct circumstances, but it loses some of its charm when executed in a completely virtual setting. The virtual setup still involves two programmers looking at the same screen and talking out their designs, but there are often logistical issues to deal with, including lag, microphone set up issues, workspace and personal considerations, and many other small, individually trivial problems that worsen the experience.
+
+Virtual work patterns need to deviate from the in-person patterns we are accustomed to. Pair programming at its core is based on the following principles:
+
+1. Generating clarity through communication
+2. Producing higher quality through collaboration
+3. Creating ownership through equal contribution
+
+Pair programming is simply one way to achieve these results. We present Red Team Testing (RTT) as a different programming method that takes the same principles but uses some of the advantages that virtual work methods provide.
```

Comment:
I would remove the "we present" which makes it a bit too much like an announcement. I would consider changing "different" with "alternate"

Reply:
The reason I want to insist on alternate is because I don't think this would be a good fit for ALL teams, it's an option to choose from :)